src/configs/QueryRules.ts

```diff
@@ -53,3 +54,9 @@ export const queryRules = {
     }
   ]
 }
+
+export const exhaustRules = {
+  "common": {
```

Comment:
Not required @pveleneni  - We can just define the exhaust config object and define all the default values